forgerock-authenticator/src/main/java/org/forgerock/android/auth/Account.java

```diff
@@ -208,6 +267,9 @@ public static Account deserialize(String jsonString) {
                     .setImageURL(jsonObject.has("imageURL") ? jsonObject.getString("imageURL") : null)
                     .setBackgroundColor(jsonObject.has("backgroundColor") ? jsonObject.getString("backgroundColor") : null)
                     .setTimeAdded(jsonObject.has("timeAdded") ? getDate(jsonObject.optLong("timeAdded")) : null)
+                    .setPolicies(jsonObject.has("policies") ? jsonObject.getString("policies") : null)
```

Comment:
May need to pay attention to this, the underlining may be a JSONObject, but we convert it to String.

Reply:
We want this treated as JSON String to make it easy to serialize to other platforms.

Reply:
Sounds good